text/0000-deprecation-staging.md

```diff
@@ -0,0 +1,652 @@
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
```

Comment:
Can you rename the file to `text/0649-deprecation-staging.md`?

Reply:
done and done

Reply:
Thanks!